python/lsst/ctrl/ingestd/rseButler.py

```diff
@@ -71,11 +71,23 @@ def ingest(self, entries: list):
             LOGGER.debug(f"adding {data_type=}, {entry=}")
             data_type_dict[data_type].append(entry)
 
+        if DataType.ZIP_FILE in data_type_dict:
+            self._ingest_zip(data_type_dict[DataType.ZIP_FILE])
         if DataType.RAW_FILE in data_type_dict:
             self._ingest(data_type_dict[DataType.RAW_FILE], "direct", True)
         if DataType.DATA_PRODUCT in data_type_dict:
             self._ingest(data_type_dict[DataType.DATA_PRODUCT], "auto", False)
 
+    def _ingest_zip(self, entries: list):
+        # zip_files = [e.file_to_ingest for e in entries]
+        zip_files = [e.get_data() for e in entries]
+        for zip_file in zip_files:
+            try:
+                self.butler.ingest_zip(zip_file)
+                LOGGER.info(f"ingested {zip_file}")
```

Comment:
[the usual spiel about f-strings in log messages] :wink:

Reply:
Doh. Fixed.